.github/workflows/actions/database-backup/action.yml

```diff
@@ -135,7 +135,7 @@ runs:
         az storage blob upload --account-name ${{ env.storage_account }} --container-name database-backup \
         --file ${{ env.BACKUP_FILE_NAME }}.tar.gz --name ${{ env.BACKUP_FILE_NAME }}.tar.gz --overwrite \
         --account-key '${{ env.STORAGE_CONN_STR }}'
-        rm ${BACKUP_FILE_NAME}.tar.gz
+        rm ${{ env.BACKUP_FILE_NAME }}.tar.gz
 
     - name: Disk cleanup
```

Comment:
The 'Disk cleanup' step isn't relevant here. I used it on register to clear space on the disk before creating the backups as there wasn't enough space on the runner.